exporters/committime/collector_base.py

```diff
@@ -106,17 +105,8 @@ def get_metrics_from_apps(self, apps, namespace):
             # assume for now that there will only be one repo/branch per app
             # For jenkins pipelines, we need to grab the repo data
             # then find associated s2i/docker builds from which to pull commit & image data
-            repo_url = None
-            if jenkins_builds:
-                # we will default to the repo listed in '.spec.source.git'
-                repo_url = jenkins_builds[0].spec.source.git.uri
-
-                # however, in cases where the Jenkinsfile and source code are separate, we look for params
-                git_repo_regex = re.compile(r"(\w+://)(.+@)*([\w\d\.]+)(:[\d]+){0,1}/*(.*)")
-                for env in jenkins_builds[0].spec.strategy.jenkinsPipelineStrategy.env:
-                    result = git_repo_regex.match(env.value)
-                    if result:
-                        repo_url = env.value
```

Comment:
Why is this code being removed?  It seems valid:  https://github.com/redhat-cop/pelorus/blob/master/exporters/committime/collector_base.py#L109-L119